app/controllers/verify/phone_controller.rb

```diff
@@ -9,7 +9,10 @@ def new
     end
 
     def create
-      if step.complete
+      result = step.submit
+      analytics.track_event(Analytics::IDV_PHONE_CONFIRMATION, result.to_h)
```

Comment:
Ahh. My bad. I thought you meant moving the entire method definition in track event, not just the call to `to_h`. Derp!

The problem is that sometimes we need the result in the controller to already be a Hash, so that we can merge in more stuff, or remove stuff, like you have done in your OpenId Connect work (e.g. `result.except(:redirect_url)`)

Since `Analytics#track_event` always expects the attributes argument to be a Hash, I think it makes sense to always pass a Hash to it, as opposed to sometimes passing a Hash, and sometimes passing an object that happens to respond to `to_h`. Thoughts?